packages/glimmer-runtime/lib/compiled/opcodes/content.ts

```diff
@@ -13,6 +13,21 @@ export function normalizeTextValue(value: Opaque): string {
   }
 }
 
+export function normalizeTextOrTrustedValue(value: Opaque): Opaque {
+  if (isSafeString(value)) {
+    return value;
+  } else {
+    return normalizeTextValue(value);
+  }
+}
+
+export function isSafeString(value: Opaque): boolean {
+  // FIXME: This feels not good enough to detect SafeStrings. What is the value
```

Comment:
The implementation here (checking `toHTML`) is correct. We do not have the option to change what it means to be a `SafeString` at this point.  Changing would break a number of things elsewhere.

Can you remove the inline code comment here suggesting it is incorrect?